docs/source/dataset.rst

```diff
@@ -0,0 +1,232 @@
+.. _dataset:
+
+Dataset
++++++++
+
+The dataset configuration can be found :ref:`here <configurationdefaultparam>`, this file contain 
+specific information about input data parameters for the execution of your command. The documentation
+on the parameters use is explain in the :ref:`yaml <yamlparameters>` section.
+
+The sampling and inference steps requires a csv referencing input data. An example of input csv for
+massachusetts buildings dataset can be found in 
+`tests <https://github.com/NRCan/geo-deep-learning/blob/develop/tests/tiling/tiling_segmentation_binary_ci.csv>`_. 
+Each row of this csv is considered, in geo-deep-learning terms, to be an 
+`AOI <https://torchgeo.readthedocs.io/en/latest/user/glossary.html#term-area-of-interest-AOI>`_.
+
+.. list-table:: **Example CSV**
+   :header-rows: 1
+
+   * - raster path 
+     - vector ground truth path
+     - dataset split
+     - aoi id (optional)
+   * - my_dir/my_geoimagery1.tif
+     - my_dir/my_geogt1.gpkg
+     - trn
+     - Ontario-1
+   * - my_dir/my_geoimagery2.tif
+     - my_dir/my_geogt2.gpkg
+     - tst
+     - NewBrunswick-23
+   * - ...
+     - ...
+     - ...
+     - ...
+
+.. note::
+    
+     If left blank, the aoi id will be derived from the raster path and bands requested.
+
+The path to a custom csv must be entered in the 
+`dataset configuration <https://github.com/NRCan/geo-deep-learning/blob/develop/config/dataset/test_ci_segmentation_binary.yaml>`_.
+See the :ref:`yaml <yamlparameters>` section for more information.
+
+Dataset splits
+--------------
+
+Split in csv should be either "trn", "tst" or "inference". The validation split is automatically 
+created during tiling. Its proportion is set by the :ref:`tiling config <datatiling>`.
+
+Raster and vector file compatibility
+------------------------------------
+
+Rasters to be used must be in a format compatible with 
+`rasterio <https://rasterio.readthedocs.io/en/latest/quickstart.html?highlight=supported%20raster%20format#opening-a-dataset-in-reading-mode>`_ 
+and `GDAL <https://gdal.org/drivers/raster/index.html>`_ (ex.: GeoTiff). Same for the labels 
+(aka annotations) for each image must be stored as polygons in a `Geopandas <https://geopandas.org/en/stable/docs/user_guide/io.html>`_ 
+compatible vector file.
+
+.. _datasetsinglemultiband:
+
+Single-band vs multi-band imagery
+---------------------------------
+
+Remote sensing is known to deal with raster files from a wide variety of formats and flavors. 
+To provide as much 
+flexibility as possible with variable input formats for raster data, geo-deep-learning supports:
+1. Multi-band raster files, to be used as is (all bands needed, all bands is expected order)

```

Comment:
This section does not appear as point form in the doc: 
![image](https://user-images.githubusercontent.com/38693210/223553120-6d01e819-258e-4c5f-ba52-2767d95412c1.png)